experimental/Schemes/Types.jl

```diff
@@ -638,6 +625,29 @@ identifications given by the glueings in the `default_covering`.
     function production_func(F::AbsPreSheaf, U::PrincipalOpenSubset)
       haskey(ID, U) && return ID[U]
       V = ambient_scheme(U)
+      # In case the ambient_scheme is leading out of the admissible domain, 
+      # this is a top-chart and we have to reconstruct from below.
```

Comment:
what is the admissible domain? I know what an admissible chart is. Is it something else?

Reply:
An admissible chart is one for which `is_open_func(F)(U)` returns `true`.  This then falls into the general lookup and construction pattern documented already for the modules.

Reply:
Yes. I know that. But then call it an admissible chart and not an admissible domain.
The term "admissible domain" is not yet defined.

Reply:
A "chart" is a patch of the `default_covering` of a scheme. Here "domain" is referring rather colloquially to the "domain of all admissible input". But I agree that comment has not been very well formulated.